src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,124 @@
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try {
+            String query = "INSERT INTO manufacture (name, country, isexist) VALUES (?, ?, ?)";
+            PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection()
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setBoolean(3, true);
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            resultSet.close();
+            preparedStatement.close();
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        Manufacturer manufacturer = null;
+        String query = "SELECT * FROM manufacture WHERE id = ? AND isexist = true";
+        try {
+            PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection().prepareStatement(query);
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+                manufacturer.setName(resultSet.getObject(2, String.class));
+                manufacturer.setCountry(resultSet.getObject(3, String.class));
+            }
+            resultSet.close();
+            preparedStatement.close();
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException ex) {
+            System.out.println("Connection failed..." + ex);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        Manufacturer manufacturer = null;
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String query = "SELECT * FROM manufacture WHERE isexist = true";
+        try {
+            PreparedStatement preparedStatement
+                    = ConnectionUtil.getConnection().prepareStatement(query);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = new Manufacturer();
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+                manufacturer.setName(resultSet.getObject(2, String.class));
+                manufacturer.setCountry(resultSet.getObject(3, String.class));
+                manufacturers.add(manufacturer);
+            }
+            resultSet.close();
```

Comment:
it's a bit redundant to close result set. If statement is closed correctly, resultSet will be closed as well
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()

Reply:
done